config/locales/publications/itt_monthly_report_generator.yml

```diff
@@ -0,0 +1,23 @@
+en:
+  publications:
+    itt_monthly_report_generator:
+      candidate_headline_statistics:
+        title: Candidate Headline statistics
+        this_cycle: This cycle
+        last_cycle: Last cycle
```

Comment:
I am not using this_cycle, last_cycle translation but @avinhurry you can use once is merged